www_src/components/color-group/color-group.jsx

```diff
@@ -0,0 +1,50 @@
+var React = require('react/addons');
+var Link = require('../link/link.jsx');
+var classNames = require('classnames');
+
+var ColorGroup = React.createClass({
+  colors: ['transparent', '#FFF', '#99CA47', '#EFC246', '#E06A2C', '#69A0FC'],
+  mixins: [React.addons.LinkedStateMixin],
+  getDefaultProps: function () {
+    return {
+      id: 'value',
+      tinkerUrl: '/projects/123/elements/1/color'
+    };
+  },
+  getInitialState: function () {
+    return {
+      value: 'transparent'
+    };
+  },
+  render: function () {
+    var linkState = this.props.linkState || this.linkState;
+    var valueLink = linkState(this.props.id);
+    function onChange(e) {
```

Comment:
Seems weird to have this be in the render function. I'd make it a method instead...